components/activity_log_modal/activity_log_modal.jsx

```diff
@@ -138,8 +116,8 @@ export default class ActivityLogModal extends React.Component {
     render() {
         const activityList = [];
 
-        for (let i = 0; i < this.state.sessions.length; i++) {
-            const currentSession = this.state.sessions[i];
+        for (let i = 0; i < this.props.sessions.length; i++) {
+            const currentSession = this.props.sessions[i];
             const lastAccessTime = new Date(currentSession.last_activity_at);
```

Comment:
Can we have a smaller component to deal with each session log?  Name can be like `ActivityLog`.